processing/src/main/java/org/apache/druid/segment/nested/NestedFieldColumnIndexSupplier.java

```diff
@@ -1204,6 +1256,9 @@ private abstract class NestedVariantIndexes
     final FrontCodedIntArrayIndexed arrayDictionary = globalArrayDictionarySupplier == null
                                                       ? null
                                                       : globalArrayDictionarySupplier.get();
+    // For every single String value, we need to look up indexes from stringDictionary, longDictionary and
+    // doubleDictionary. Hence, the compute cost for one value is 3.
+    static final int INDEX_COMPUTE_SCALE = 3;
```

Comment:
i actually think 1 would probably be ok here too since we still only use a single bitmap, but this is also fine